apps/crc_interactive.py

```diff
@@ -115,7 +121,9 @@ def create_srun_command(self, args: Namespace) -> str:
             'feature': '--constraint={}',
             'num_cores': '--cpus-per-task={}' if getattr(args, 'openmp') else '--ntasks-per-node={}'
         }
-
+        #Check if time specified in %H:%M format
```

Comment:
Duplicate logic. I suggest adding a static method for casting the argument from a string into an integer. Then, add some tests to support the new method.

Reply:
This was done to pass on the correct time argument to srun via the formatting dictionary. I removed the seconds specification in the dictionary which should now work for both integer hours and %H:%M time format.